sw/device/lib/dif/dif_otbn.h

```diff
@@ -26,9 +26,9 @@ extern "C" {
  * OTBN commands
  */
 typedef enum dif_otbn_cmd {
-  kDifOtbnCmdExecute = 0x01,
-  kDifOtbnCmdSecWipeDmem = 0x02,
-  kDifOtbnCmdSecWipeImem = 0x03,
+  kDifOtbnCmdExecute = 0xd8,
+  kDifOtbnCmdSecWipeDmem = 0xc3,
+  kDifOtbnCmdSecWipeImem = 0x1e,
```

Comment:
The same change needs to be made in `sw/device/silicon_creator/lib/drivers/otbn.h` and in `hw/ip/otbn/dv/otbnsim/sim/constants.py`.